src/freenas/usr/local/libexec/freenas-debug/hardware/hardware.sh

```diff
@@ -211,6 +211,14 @@ hardware_freebsd()
 		sas3flash -listall
 		section_footer
 	fi
+
+	if case "$(midclt call truenas.get_chassis_hardware)" in TRUENAS-M*) true;; *) false;; esac; then
```

Comment:
Why this complicated if case and not `if ["$(midclt call truenas.get_chassis_hardware)" = "TRUENAS-M" ]; then`